app/client.ts

```diff
@@ -0,0 +1,47 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export interface Report {
+  reporterFid: bigint;
+  sybilFid: bigint;
+  castHash: string | undefined;
+  messageHash: string | undefined;
+  network: number | undefined;
+  reportTimestamp: string | undefined;
+}
+
+export async function createReport(report: Report) {
+  const _report = await prisma.report.create({
+    data: report,
+  });
+  return _report;
+}
+
+export async function getSybilReportCount(fid: bigint) {
+  const reportCount = await prisma.report.findMany({
```

Comment:
this is doing an unnecessary query. In this case you are returning all the reports and getting the length of the response array. The most interesting thing would be to directly use the SQL COUNT clause e.g https://www.prisma.io/docs/orm/prisma-client/queries/aggregation-grouping-summarizing#count